### PR TITLE
CircleCI configs / Tidy up pom.xml / README.md badges 

### PR DESCRIPTION
**Pull request by pnsantos**
Similar to https://github.com/dockerunit/dockerunit-core/pull/5#issue-284090274

### CODE REVIEW

**Comment by pnsantos** [comment]
FYI: the reason Better Code Hub failed was because I ran it previously with a "wrong" config and now it things one of the metrics got worse. I'll re-enable the check after this gets merged.

File: pom.xml [hunk 1]
```diff
@@ -1,90 +1,96 @@
...
+
+  <scm>
+    <connection>scm:https://github.com/dockerunit/dockerunit-junit5.git</connection>
+    <developerConnection>scm:https://github.com/dockerunit/dockerunit-junit5.git</developerConnection>
+    <url>https://github.com/dockerunit/dockerunit-junit5.git</url>
+  </scm>
 
   <properties>
+    <lombok.version>1.18.8</lombok.version>
     <junit.version>5.4.2</junit.version>
```
Comment (DarthSeppius) [inline pom.xml:27]: probably you can also bump the version of Junit 5 to `5.5.0` if you want.

**Review by DarthSeppius — APPROVED** [review]
LGTM

**Review by irotech — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (pnsantos) [reply pom.xml:27]: I'll leave it for dependabot, for now just wanna get this in.

**Pull request merged**